doc/component_descriptor_registries.rst

```diff
@@ -18,6 +18,26 @@ URL *MUST* refer to an OCI Image Registry (for example `eu.gcr.io/gardener-proje
 
 The base URL *MAY* refer to sub-path on a OCI Image Registry.
 
+The thus-referenced `Component Descriptor` is stored as an OCI Container Image would be. The
+Image Manifest *MUST* contain exactly one layer. The layer *MUST* be a POSIX.1-2001-compliant
+pax archive (commonly also known as "tarball"), containing as first entry a regular file named
+`component-descriptor.yaml`, whose `utf-8`-encoded content is a valid `Component Descriptor` (v2),
+serialised as either YAML or JSON.
```

Comment:
> The layer *MUST* be a POSIX.1-2001-compliant
pax archive

I thought that we would also allow the use json directly?
So wtihout the need to tar it.

Reply:
@schrodit : you are correct - we(/I) never updated this documentation, apparently..

Reply:
@schrodit : you are correct - we(/I) never updated this documentation, apparently..

Reply:
will create a separate PR for this - okay?